protected/models/Patient.php

```diff
@@ -171,6 +173,7 @@ public function relations()
             'diagnoses' => array(self::MANY_MANY, '\OEModule\OphCiExamination\models\OphCiExamination_Diagnosis', 'patient_diagnosis_assignment(patient_id, id)',
                 'alias' => 'patient_diagnoses',
                 'order' => 'patient_diagnoses.principal DESC', ),
+            'systemic_diagnoses' => array(self::MANY_MANY, '\OEModule\OphCiExamination\models\SystemicDiagnoses_Diagnosis', 'patient_systemic_diagnosis(patient_id, id)'),
```

Comment:
Many to Many, is this right? Would multiple patients have the same patient-diagnosis?

Reply:
This is a relationship that doesn't follow a foreign key. I don't know if there's a way to do that properly with a 0..N relationship in Yii.